opencti-platform/opencti-graphql/src/config/conf.js

```diff
@@ -383,6 +399,15 @@ export const logSupport = {
   warn: (message, meta = {}) => logSupport._log('warn', message, null, meta),
 };
 
+const LOG_TELEMETRY = 'TELEMETRY';
+export const logTelemetry = {
+  _log: (level, message, error, meta = {}) => {
```

Comment:
I think we can remove error from here (and give null directely in addBasicMetaInformation below)